src/Sentry/SentryClient.cs

```diff
@@ -63,16 +63,15 @@ public SentryClient(SentryOptions options)
         }
 
         /// <summary>
-        /// Queues the event to be sent to Sentry
+        /// Queues the event to be sent to Sentry.
         /// </summary>
         /// <remarks>
         /// An optional scope, if provided, will be applied to the event.
         /// </remarks>
         /// <param name="event">The event to send to Sentry.</param>
         /// <param name="scope">The optional scope to augment the event with.</param>
-        /// <returns></returns>
         /// <inheritdoc />
-        public SentryId CaptureEvent(SentryEvent @event, Scope scope = null)
+        public SentryId CaptureEvent(SentryEvent? @event, Scope? scope = null)
```

Comment:
Event shouldn't really be nullable (even though the code checks for it below to avoid NPE).
I'll address it in a follow up PR